src/BrandEmbassyCodingStandard/Sniffs/Classes/ForbiddenStaticMethodSniff.php

```diff
@@ -28,16 +29,18 @@ final class ForbiddenStaticMethodSniff implements Sniff
      */
     public $forbiddenClasses = [];
```

Comment:
Ony to vlastně nejsou forbidden classy. Jsou to classy, jejíchž metody se nemí využívat pomocí `self` nebo `static`. Resp. classy, které mají statické metody, které je třeba volat napřímo.

Zasrané pojmenovávání věcí...